index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+		<link rel="stylesheet" href="style.css">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
+		
+
+</head>
+<body>
+	<header id="ToolBar">
+		<img src="Images\Logo Placeholder.svg" alt="Logo" id="logo-placeholder">
+
+		<a href="#" id="tool-Button">
+			<img src="Images\MenuIcon.svg" alt="Menu Icon">		
+		</a>
```

Comment:
- [ ] I would suggest you please wrap this hamburger button with a `nav` tag for semantics reason and the right position for convention and user experience for mobile devices. Also, you will have a nav link while working on the desktop version so It would be easy to put link inside the `nav` tag for semantic reason